tests/integration/test_zookeeper_config_load_balancing/configs/zookeeper_load_balancing2.xml

```diff
@@ -0,0 +1,29 @@
+<clickhouse>
+    <zookeeper>
+        <!--<zookeeper_load_balancing> random / in_order / nearest_hostname / first_or_random / round_robin </zookeeper_load_balancing>-->
+        <zookeeper_load_balancing>random</zookeeper_load_balancing>
+
+        <client_availability_zone>az2</client_availability_zone>
```

Comment:
cc @thevar1able  We already have a configuration section on this at server level. it would be better UX to only need to define this once?

https://github.com/ClickHouse/ClickHouse/pull/59976/files#diff-47e5b6a4c49f50e47acd19ca0765ddd7940da8d3d4e7e846bc25a3f0070e0399R3

Reply:
Indeed. We should reuse `PlacementInfo` here.

Reply:
We initialize `PlacementInfo` too late:
https://github.com/ClickHouse/ClickHouse/blob/784f66cf2f77f9dc83e193c61b66e64b3e03624b/programs/server/Server.cpp#L1809
ZooKeeper can be initialized here: 
https://github.com/ClickHouse/ClickHouse/blob/784f66cf2f77f9dc83e193c61b66e64b3e03624b/programs/server/Server.cpp#L1011-L1015

I will move it earlier, but it will not be possible to use zk includes for PlacementInfo

Reply:
Also, we still need a setting that enables/disables az-aware balancing in ZooKeeper client